doc/source/train/user-guides/data-loading-preprocessing.rst

```diff
@@ -177,25 +185,123 @@ Using Ray Data and Ray Train for distributed training on large datasets involves
             trainer.fit()
 
 
-Migrating from PyTorch DataLoader
----------------------------------
+.. _train-datasets-load:
+
+Loading data
+~~~~~~~~~~~~
+
+Ray Datasets can be created from many different data sources and formats. For more details, see :ref:`Loading Data <loading_data>`.
+
+.. _train-datasets-preprocess:
+
+Preprocessing data
+~~~~~~~~~~~~~~~~~~
+
+Ray Data support a wide range of preprocessing operations that can be used to transform your data prior to training.
+
+- For general preprocessing, see :ref:`Transforming Data <transforming_data>`.
+- For tabular data, see :ref:`Preprocessing Structured Data <preprocessing_structured_data>`.
+- For PyTorch tensors, see :ref:`Transformations with torch tensors <transform_pytorch>`.
+- For optimizing expensive preprocessing operations, see :ref:`Caching the preprocessed dataset <dataset_cache_performance>`.
+
+.. _train-datasets-input:
+
+Inputting and splitting data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Your preprocessed datasets can be passed into a Ray Train Trainer (e.g. :class:`~ray.train.torch.TorchTrainer`) through the ``datasets`` argument.
+
+The datasets passed into the Trainer's ``datasets`` can be accessed inside of the ``train_loop_per_worker`` run on each distributed training worker by calling :meth:`ray.train.get_dataset_shard`. 
```

Comment:
<img width="793" alt="Screenshot 2023-08-17 at 10 34 09 AM" src="https://github.com/ray-project/ray/assets/26745457/6b0765a5-a469-499d-955e-2e398002bee4">

Nit: seems to have too many hyperlinks here, in a single paragraph, shall we only link the api for the first time, and use ```` in the following occurrences?

Reply:
@amogkam lmk your opinion here, another option would be to shorten to `get_dataset_shard()` and keep the link.